elife/nodejs6.sls

```diff
@@ -13,3 +13,20 @@ nodejs6:
         - name: nodejs
         - require:
             - pkgrepo: nodejs6
+
```

Comment:
[On 18.04](https://alfred.elifesciences.org/blue/rest/organizations/jenkins/pipelines/pull-requests-projects/pipelines/builder-base-formula/branches/PR-209/runs/1/nodes/31/steps/43/log/?start=0), the previous state is installing `nodejs` 8 rather than 6:
```
2019-03-15T04:38:43.560Z] [54.163.0.155] out:           ID: nodejs6
[2019-03-15T04:38:43.560Z] [54.163.0.155] out:     Function: pkg.latest
[2019-03-15T04:38:43.560Z] [54.163.0.155] out:         Name: nodejs
[2019-03-15T04:38:43.560Z] [54.163.0.155] out:       Result: True
[2019-03-15T04:38:43.560Z] [54.163.0.155] out:      Comment: The following packages were successfully installed/upgraded: nodejs
[2019-03-15T04:38:43.560Z] [54.163.0.155] out:      Started: 04:36:03.714207
[2019-03-15T04:38:43.560Z] [54.163.0.155] out:     Duration: 5172.968 ms
...
[2019-03-15T04:38:43.560Z] [54.163.0.155] out:               nodejs:
[2019-03-15T04:38:43.560Z] [54.163.0.155] out:                   ----------
[2019-03-15T04:38:43.560Z] [54.163.0.155] out:                   new:
[2019-03-15T04:38:43.560Z] [54.163.0.155] out:                       8.10.0~dfsg-2ubuntu0.4
[2019-03-15T04:38:43.560Z] [54.163.0.155] out:                   old:
```